gitlab_api.py

```diff
@@ -1,12 +1,21 @@
+import base64
+from collections import namedtuple
 from datetime import datetime, timedelta
 
 import requests
+import urllib3
+import yaml
 from flask import flash
 
 import config
 import routes.overview_page
 from utils import get_repos_info, load_json_data, save_json_data_and_return
 
+urllib3.disable_warnings()
```

Comment:
It would be nice to have a comment above this line to record what is this necessary for. The reason might disappear in the future and you won't know whether you can remove this or not.